### PR TITLE
docs(trace): updated conventions for submodel descriptors and EDC asset structure

### PR DESCRIPTION
**Pull request by markuskeidl**
Updated conventions for submodel descriptors and EDC asset structure structure to give data provides more flexibility in how to create EDC assets for submodels of digital twins based on the new AAS v3.0 standard

### CODE REVIEW

**Comment by markuskeidl** [comment]
Merge conflicts fixed - pull request is finished (although there will be changes in the upcoming weeks, I think). But I think it makes sense to add it to the 3.2 KIT to have a somewhat consistent version for implementation @mhellmeier

File: docs/kits/Traceability Kit/page_changelog.md [hunk 1]
```diff
@@ -11,6 +11,20 @@ sidebar_position: 1
...
+- ./.
+
+### Changed
+
+- Development View: Updated conventions for submodel descriptors and EDC asset structure to give data provides more flexibility in how to create EDC assets for submodels of digital twins
+
+### Removed
+
+- ./.
+
```
Comment (mhellmeier) [inline docs/kits/Traceability Kit/page_changelog.md:27]: Thanks for updating the changelog! There are still some other pull requests already merged into the `docs/trace-release-3.2` branch. Would it be possible to mention the already merged changes here? In the future, we should constantly update the changelog in every merge request (missed in the early ones).

**Review by mhellmeier — CHANGES_REQUESTED** [review]

File: docs/kits/Traceability Kit/Software Development View/page_software-development-view.md [hunk 2]
```diff
@@ -411,79 +411,149 @@ The lookup for parts can use the customerPartId or the manufacturerPartId. Both,
...
-        "subprotocolBody": "asset:prop:id=123;idsEndpoint=http://edc.control.plane/",
+        "href": "https://edc.data.plane/{path}",
+        "endpointProtocol": "HTTP",
+        "endpointProtocolVersion": ["1.1"],         
+        "subprotocol": "IDS",
+        "subprotocolBody": "id=123;idsEndpoint=http://edc.control.plane/",
+        "subprotocolBodyEncoding": "plain",
+        "securityAttributes": [ 
+          { "type": "NONE", "key": "NONE", "value": "NONE" }
+        ]
```
[the remark this answers is not in the document]
Comment (markuskeidl) [reply docs/kits/Traceability Kit/Software Development View/page_software-development-view.md]: Whitespace issues should be fixed now.

File: docs/kits/Traceability Kit/Software Development View/page_software-development-view.md [hunk 3]
```diff
@@ -411,79 +411,149 @@ The lookup for parts can use the customerPartId or the manufacturerPartId. Both,
...
+        "protocolInformation": {
+          "href": "https://edc.data.plane/urn%3Auuid%3A75e98d67-e09e-4388-b2f6-ea0a0a642bfe-urn%3Auuid%3A7effd7f4-6353-4401-9547-c54b420a22a0",
+          "endpointProtocol": "HTTP",
+          "endpointProtocolVersion": ["1.1"],         
+          "subprotocol": "IDS",
+          "subprotocolBody": "id=urn:uuid:75e98d67-e09e-4388-b2f6-ea0a0a642bfe-urn:uuid:7effd7f4-6353-4401-9547-c54b420a22a0;idsEndpoint=http://edc.control.plane/",
+          "subprotocolBodyEncoding": "plain",
+          "securityAttributes": [ 
+            { "type": "NONE", "key": "NONE", "value": "NONE" }
+          ]
```
[the remark this answers is not in the document]
Comment (markuskeidl) [reply docs/kits/Traceability Kit/Software Development View/page_software-development-view.md]: Whitespace issues should be fixed now.

File: docs/kits/Traceability Kit/Software Development View/page_software-development-view.md [hunk 4]
```diff
@@ -411,79 +411,149 @@ The lookup for parts can use the customerPartId or the manufacturerPartId. Both,
...
-          "subprotocolBody": "asset:prop:id=urn:uuid:1475f313-0a83-4e2b-b705-a100eebcb7d7;idsEndpoint=http://control-plane.edc.catena-x.net/"
+          "href": "https://edc.data.plane/urn%3Auuid%3A75e98d67-e09e-4388-b2f6-ea0a0a642bfe-urn%3Auuid%3A7effd7f4-6353-4401-9547-c54b420a22a0",
+          "endpointProtocol": "HTTP",
+          "endpointProtocolVersion": ["1.1"],         
+          "subprotocol": "IDS",
+          "subprotocolBody": "id=urn:uuid:1475f313-0a83-4e2b-b705-a100eebcb7d7;idsEndpoint=http://control-plane.edc.catena-x.net/",
+          "subprotocolBodyEncoding": "plain",
+          "securityAttributes": [ 
+            { "type": "NONE", "key": "NONE", "value": "NONE" }
+          ]
```
[the remark this answers is not in the document]
Comment (markuskeidl) [reply docs/kits/Traceability Kit/Software Development View/page_software-development-view.md]: Whitespace issues should be fixed now.

File: docs/kits/Traceability Kit/page_changelog.md (hunk 1 above)
Comment (markuskeidl) [reply docs/kits/Traceability Kit/page_changelog.md]: Updated changelog with missing messages.

**Comment by markuskeidl** [comment]
Fixed all merge conflicts and resolved all issues.

**Review by mhellmeier — APPROVED** [review]
LGTM

**Pull request merged**